cmd/gateway-gcs-layer.go

```diff
@@ -197,10 +218,28 @@ func (l *gcsGateway) MakeBucket(bucket string) error {
 func (l *gcsGateway) MakeBucketWithLocation(bucket, location string) error {
 	bkt := l.client.Bucket(bucket)
 
-	// TODO: location
-	_ = location
+	// this will map s3 regions to google multi regions
+	if v, ok := map[string]string{
```

Comment:
Maybe just look at the first part of the AWS region, i.e. all 'ap's to asia, 'eu's to eu, etc.
This way it is a bit more resilient against future addtions of AWS regions.